Sources/ZcashLightClientKit/Rust/ZcashRustBackend.swift

```diff
@@ -286,7 +285,8 @@ struct ZcashRustBackend: ZcashRustBackendWelding {
         accountUUID: AccountUUID,
         to address: String,
         value: Int64,
-        memo: MemoBytes?
+        memo: MemoBytes?,
+        confirmationsPolicy: ConfirmationsPolicy = ConfirmationsPolicy.defaultTransferPolicy()
```

Comment:
We do not set default values in the internal Rust backend APIs. Remove all of them from `ZcashRustBackend`:
```suggestion
        confirmationsPolicy: ConfirmationsPolicy
```
(and similarly below).